lib/flash_flow/release/percy_client.rb

```diff
@@ -0,0 +1,67 @@
+require 'percy'
+
+module FlashFlow
+  module Release
+    class PercyClient
+
+      def initialize(config={})
+        @client = initialize_connection!(config['token'])
+      end
+
+      def find_latest_by_sha(sha)
+        response = get_builds
+        commit = find_commit_by_sha(response, sha)
+        build = find_build_by_commit_id(response, commit['id'])
+
+        { url: build['web-url'], approved: !build['approved-at'].nil? }
+      end
+
+      private
+
+      def initialize_connection!(token)
+        if token.nil?
+          raise RuntimeError.new("Percy token must be set in your flash flow config.")
+        end
+
+        Percy.client.config.access_token = token
+        percy_client
+      end
+
+      def percy_client
+        Percy.client
+      end
+
+      def get_builds
```

Comment:
This could probably be moved to the percy ruby gem, but for now it's fine here since this api isn't officially supported.